test/test_torch.py

```diff
@@ -1887,6 +1887,32 @@ def _test_gather(self, cast, test_bounds=True):
         def test_gather(self):
             self._test_gather(self, lambda t: t)
 
+        def test_ravel(self):
```

Comment:
You should add an entry to https://github.com/pytorch/pytorch/blob/59414b359d3a624c63032140e0e1fbc654aa6f5f/torch/testing/_internal/common_methods_invocations.py#L593 to auto-generate an autograd test. You'll be able to see the autograd test by running `pytest test/test_autograd.py -v -k "ravel"` locally (the -k "ravel" tells pytest to filter tests by name)